frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -348,15 +348,15 @@ export class OverviewPageComponent extends React.Component<OverviewProps, State>
       });
   }
 
-  fetchHealthChunk(chunk: NamespaceInfo[], duration: DurationInSeconds, type: OverviewType) {
+  async fetchHealthChunk(chunk: NamespaceInfo[], duration: DurationInSeconds, type: OverviewType) {
```

Comment:
No, it is not necessary. I forgot to mention it in the refactoring bonus section of the PR. Code editor did the suggestion since it returns a promise and I just add the word for cleaner syntax.

Reply:
It was OK, I questioned this one mainly because you also had to update the body of the function.  In other places you could add `async` and the function was otherwise untouched.